opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -104,10 +104,21 @@ enum SeverityNumber {
   SEVERITY_NUMBER_FATAL4 = 24;
 }
 
-// Masks for LogRecord.flags field.
+// LogRecordFlags is defined as a protobuf 'uint32' type and is to be used as
+// bit-fields. Each non-zero value defined in this enum is a bit-mask.
+// To extract the bit-field, for example, use an expression like:
+//
+//   (logRecord.flags & LOG_RECORD_FLAG_TRACE_FLAGS_MASK)
+//
 enum LogRecordFlags {
```

Comment:
Having a "Mask" in all the values (by having it in the enum and prefix all values with enum name as we do everywhere else).

Reply:
I renamed FLAG_NO_RECORDED_VALUE -> FLAG_NO_RECORDED_VALUE_MASK. Now all enum elements have the _MASK suffix.

Do you also want to have the Mask suffix for the enum names (in addition to enum element names)?